packages/block-editor/src/components/block-list/use-block-props/use-selected-block-event-handlers.js

```diff
@@ -21,12 +24,16 @@ import { unlock } from '../../../lock-unlock';
  * @param {string} clientId Block client ID.
  */
 export function useEventHandlers( { clientId, isSelected } ) {
```

Comment:
This hook is becoming a bit of a dump every even hook. We should consider a better organization here (maybe a follow-up)